default-config.sh

```diff
@@ -25,7 +25,7 @@ METADATA_FILE_NAME=metadata.sh
 # simply not use that syntax and override, since these variables are
 # read last.
 WORKER_RMM_POOL_SIZE=${WORKER_RMM_POOL_SIZE:-12G}
-DASK_CUDA_INTERFACE=${DASK_CUDA_INTERFACE:-ib0}
+DASK_CUDA_INTERFACE=${DASK_CUDA_INTERFACE:-ibp5s0f0}
```

Comment:
I believe this is Draco-specific, but that's probably necessary...unless if there's some sort of common alias or something that can be used.  This is fine though.